apps/api/migrations/20200328100536_model-part-2.js

```diff
@@ -0,0 +1,47 @@
+exports.up = function (knex) {
+    return knex.schema
+        .createTable('request', function (table) {
+            table
+                .uuid('id')
+                .primary()
+                .defaultTo(knex.raw('uuid_generate_v4()'));
+            table.string('requester_name').notNullable();
+            table.enu(
+                'requester_type',
+                ['finess', 'rpps', 'adeli', 'other'],
+                { useNative: true, enumName: 'health_professional_type' }
+            ).notNullable();
+            table.string('requester_professional_identifier').notNullable();
+            table.string('contact_name').notNullable();
+            table.string('contact_email').notNullable();
+            table.string('contact_phone').notNullable();
+            table.text('delivery_address').notNullable();
+            table.string('delivery_postal_code').notNullable();
+            table.string('delivery_city').notNullable();
+            table.text('requester_comment').nullable();
+            table.integer('mask_small_size_quantity').notNullable().defaultTo(0);
+            table.integer('mask_large_size_quantity').notNullable().defaultTo(0);
+            table.uuid('id_production_management').nullable();
```

Comment:
`production_management_id` would be a lot more english fluent IMO

Reply:
To be consistent, we need to change also `ids` field on `production_management_user` table